util/test/rdtest/shared/Discard_Zoo.py

```diff
@@ -101,6 +101,12 @@ def check_texture(self, id, discarded: bool):
                         for (x, y) in [(0, 0), (1, 0), (2, 0), (3, 0), (4, 0), (5, 0), (1, 1), (2, 2), (3, 3),
                                        (4, 4), (w - 1, h - 1), (w - 2, h - 2), (w - 1, 0), (w - 1, 1), (w - 1, 2),
                                        (0, h - 1), (1, h - 1), (2, h - 1), (3, h - 2)]:
+                            # overflow can happen with 1D textures or tiny mips
+                            if x >= w:
+                                x = w-1
+                            if y >= h:
+                                y = h-1
```

Comment:
This should not be necessary, this test is already passing on many drivers and picking pixels should return clamped results.